src/main/kotlin/no/nav/syfo/db/UtsendtVarselDAO.kt

```diff
@@ -1,7 +1,7 @@
 package no.nav.syfo.db
 
 import no.nav.syfo.db.domain.PPlanlagtVarsel
-import no.nav.syfo.db.domain.UtsendtVarsel
+import no.nav.syfo.db.domain.PUtsendtVarsel
 import java.sql.Timestamp
 import java.time.LocalDateTime
 import java.util.*
```

Comment:
Hvordan er det med den gamle
```
fun DatabaseInterface.storeUtsendtVarsel(planlagtVarsel: PPlanlagtVarsel) {
...
}
```
som fortsatt kalles fra utsending av planlagte varsler? Vil den nye utsendt_varsel tabellen
brekke SQL'en som kjøres da siden det mangler noen kolonner (kanal, orgnummer, narmesteLederFnr)?

Reply:
Fint at du nevner det, for det hadde jeg tenkt til å sjekke, men så glemte jeg det visst. I'm on it.

Reply:
De er nullable, så det skal gå bra, men man vet jo aldri..

Reply:
Ha testet i dev-fss, siden man ikke sender varsler fra dev-gcp, og det fungerte fint der